### ISSUE TITLE
[Documentation][Web][Usability] Documentation is too wide and requires scrolling

### ISSUE THREAD

**Issue by torgefaehrlich** [hashicorp/terraform-provider-null#142]
### Terraform CLI and Provider Versions

Looking at documentation on the web: https://registry.terraform.io/providers/hashicorp/null/3.1.1/docs/resources/resource
No terraform binary used/involved.
Safari browser on Mac OS 
Provider version: 3.1.1

### Terraform Configuration

```terraform
https://registry.terraform.io/providers/hashicorp/null/3.1.1/docs/resources/resource
```

### Expected Behavior

I would like to be able to read the documentation without horizontal scrolling.

### Actual Behavior

No matter what scaling and/or aspect ratio I use, the central part of the documentation always requires horizontal scrolling. While at the same time there is *tons* of white space both left and right. 

### Steps to Reproduce

1. open documentation in a browser: https://registry.terraform.io/providers/hashicorp/null/3.1.1/docs/resources/resource
2. try to read the main part of the documentation (the part with the comment tags)

### How much impact is this issue causing?

Low

### Additional Information

I'm sure this can be fixed at code level. No need to involve the web designer. Just re-do the line breaks at or before column 76.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by SBGoods** [comment]
Hi @torgefaehrlich, sorry for the delay in our response. First of all, thank you for opening this issue! I did take a look at your PR #143 for this issue, and it looks like it is failing in our test pipeline. This is because this provider uses [terraform-plugin-docs](https://github.com/hashicorp/terraform-plugin-docs/) to generate docs and the changes need to be made in the resource.tf file within the example directory in order to generate the resource.md file instead of changing the markdown file directly. 

I've added a fix to the documentation to #150 and will close out this issue once that PR is merged. Thank you again, and please feel free to reach out with any questions!

**Comment by SBGoods** [comment]
Hi @torgefaehrlich, the fix for the horizontal scroll for the documentation was released with Null v3.2.0 and now it is showing up properly in the [registry](https://registry.terraform.io/providers/hashicorp/null/latest/docs/resources/resource). Thank you again for raising this issue with us!
